project/src/main/packed-sprite.gd

```diff
@@ -48,6 +48,9 @@ func set_frame_data(new_frame_data: String) -> void:
 	else:
 		_frame_src_rects = ResourceCache.get_frame_src_rects(frame_data)
 		_frame_dest_rects = ResourceCache.get_frame_dest_rects(frame_data)
+
+		if not _frame_src_rects:
+			_load_rects_from_json()
```

Comment:
Add a code and commit comment.

Reply:
Added code comment, `if the ResourceCache is unavailable (possibly during demos...`

Added commit comment, `added fallback so that...`